Documentation/operations/performance/tuning.rst

```diff
@@ -178,16 +178,18 @@ To enable the iptables connection-tracking bypass:
 
        .. code-block:: shell-session
 
-          cilium install --config install-no-conntrack-iptables-rules=true
+          cilium install \
+            --set installNoConntrackIptablesRules=true \
+            --set kubeProxyReplacement=true
 
     .. group-tab:: Helm
 
        .. parsed-literal::
 
-           helm install cilium |CHART_RELEASE| \\
-             --namespace kube-system \\
-             --set installNoConntrackIptablesRules=true \\
-             --set kubeProxyReplacement=strict
+           helm install cilium |CHART_RELEASE| \
+             --namespace kube-system \
+             --set installNoConntrackIptablesRules=true \
+             --set kubeProxyReplacement=true
```

Comment:
Unfortunately in this case escaping is required because it is part of a parsed-literal block, which AFAIU it is a requirement to use the `CHART_RELEASE` fragment.